WordPress/build.gradle

```diff
@@ -186,6 +186,14 @@ androidExtensions {
 }
 
 dependencies {
+    // Gutenberg Mobile
+    //
+    // Use a binary from Bintray by default, but can optionally be configured
+    // to use a local build via local-builds.gradle
+    implementation "$gradle.ext.gutenbergMobileBinaryPath:$gutenbergMobileVersion"
+    testImplementation "$gradle.ext.gutenbergMobileBinaryPath:$gutenbergMobileVersion"
+    debugImplementation "$gradle.ext.gutenbergMobileBinaryPath:$gutenbergMobileVersion"
+
```

Comment:
```suggestion
```

I don't think we need this dependency in the WordPress module. As far as I know Gutenberg is only enabled through the `WordPressEditor` module, so we should be safe to remove this.

I've tested this change and the editor worked as expected for multiple different blocks.